src/code/audio_heap.c

```diff
@@ -1271,7 +1271,7 @@ void AudioHeap_DiscardSampleCaches(void) {
 typedef struct {
     u32 oldAddr;
     u32 newAddr;
-    u32 size;
+    size_t size;
     u8 newMedium;
 } StorageChange;
```

Comment:
This looks suspicious. Really it would have been better to not touch the audio files at all if you don't know what things are: changing half of it is surely only going to break it more.